cypress/integration/api/user.test.ts

```diff
@@ -0,0 +1,208 @@
+import { HttpMethod } from "@lib/http-method";
+import { User } from "@prisma/client";
+import { TEST_USER } from "cypress/support/constants";
+import { StatusCodes } from "http-status-codes";
+
+describe("User API", () => {
+  beforeEach(() => {
+    cy.task("db:setup");
+  });
+
+  afterEach(() => {
+    cy.task("db:teardown");
+  });
+
+  it("should be able to get current user", () => {
+    cy.login();
+
+    cy.request(HttpMethod.GET, "/api/user").then((response) => {
+      expect(response.status).to.equal(StatusCodes.OK);
+
+      expect(Object.keys(response.body)).to.have.members([
+        "id",
+        "createdAt",
+        "updatedAt",
+        "email",
+        "name",
+        "imageUrl",
+      ]);
+      expect(response.body.id).to.be.a("string");
+      expect(response.body.createdAt).to.be.a("string");
+      expect(response.body.updatedAt).to.be.a("string");
+      expect(response.body.email).to.equal(TEST_USER.email);
+      expect(response.body.name).to.equal(TEST_USER.name);
+      expect(response.body.imageUrl).to.equal(TEST_USER.imageUrl);
+    });
+  });
+
+  it("should be able to get other users if logged in", () => {
```

Comment:
I think a similar test that tries to get user information (both for `/api/user/` and `/api/user/[id]`) while not AuthN would be worthwhile.

Reply:
Oops, it looks like its "No Content" for `/api/user/`, not UNAUTHORIZED